backend/lib/services/shoots.js

```diff
@@ -304,7 +304,7 @@ exports.info = async function ({ user, namespace, name }) {
 
   if (shoot.spec.seedName) {
     const seed = getSeed(getSeedNameFromShoot(shoot))
-    if (seed && namespace !== 'garden') {
+    if (seed) {
```

Comment:
bug not related to this line:
open a cluster terminal for a shoot cluster, that has a non-managed seed. This works for the first terminal to be created, but for the second terminal the error `terminal cannot be hosted on non-managed seed` is shown

Reply:
Fixed, missing `updateSettings` in `initialize` function